articles/build-an-outlook-clone-using-react-hooks/index.md

```diff
@@ -0,0 +1,881 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: build-an-outlook-clone-using-react-hooks
+title: React Hooks and Building Outlook Clone
+description: Complete guide where we discuss react hooks and have written optimized code following up on our earlier topic of how to build an outlook clone using React.
+author: lalithnaryan-c
+date: 2020-07-23T00:00:00-07:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/build-an-outlook-clone-using-react-hooks/hero.jpg
+    alt: outlook clone example image
+---
+Welcome to Hook part of the React series. This time we cover implementing the same outlook clone that we built earlier. But this time, we use React Hooks. The earlier codebase was repetitive and there is a scope for optimization of the entire project. To sum up the earlier attempt at building an outlook clone, we made API calls and displayed emails according to the filter requested. We also implemented local storage.
```

Comment:
Assumptions about what the reader is familiar with that could be confusing. Consider something along the lines of, "In this tutorial, we will cover the same outlook clone that we built in <name and hyperlink to the previous article>, but this time we will use React Hooks."

Reply:
Thank you for your feedback. I will incorporate those changes immediately. Any other changes that you would like to be incorporated?